README.md

```diff
@@ -1 +1,145 @@
-# Role-Based Access Control  Server
+# Role-Based Access Control Server
+
+This is a proof-of-concept (POC) version of the Ocean role-based access control (RBAC) server.
+
+## Getting started
+
+In order to start user the RBAC server:
+
+1. Clone this repository:
+
+```Bash
+git clone https://github.com/oceanprotocol/RBAC-Server.git
+cd RBAC-Server
+```
+
+2. Install the dependancies:
+
+```Bash
+npm install
+```
+
+3. Start the server
+
+```Bash
+npm run start
+```
+
+## Sending requests
+
+- Download and install [Postman](https://www.postman.com/downloads/)
+
+- Send a GET request to `http://localhost:3000`
```

Comment:
POST

Reply:
Thanks for the feedback, I've updated this